modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenuSmoke.testcase

```diff
@@ -0,0 +1,141 @@
+@component-name = "portal-platform-experience-and-design-system"
+definition {
+
+	property osgi.modules.includes = "accessibility-menu-web";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Accessibility Menu";
+	property testray.main.component.name = "Accessibility Menu";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		TestUtils.hardRefresh();
+
+		task ("Given user with admin privileges") {
+			User.firstLoginPG();
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+
+			TestUtils.hardRefresh();
+
+			HeadlessPortalInstanceAPI.cleanAllPortalInstances();
```

Comment:
This is to delete any created portal instances right? Why not include this in the else statement too?

Reply:
I added this line ahead for [that test](https://liferay.atlassian.net/browse/LPS-187918#:~:text=Given%20user%20with%20admin%20privileges%0AAnd%20new,setting%20and%20system%20setting%20AM%20is%20ON), I shouldn't have mixed up stuff.